src/component/Pagination.jsx

```diff
@@ -0,0 +1,25 @@
+// Pagination.js
+const Pagination = ({ pagebuttons, activePage, handlePageClick }) => {
+  const getButtonStyle = (number) => ({
+    backgroundColor: activePage === number ? '#2F80ED' : 'white',
+    color: activePage === number ? 'white' : 'black',
+  });
+
+  return (
+    <div className="pagination">
+      <button>&lt;</button>
```

Comment:
P2:
button 사용시 type을 명시해서 사용해주세요.

```suggestion
      <button type='button'>&lt;</button>
```